packages/synapse-react-client/src/utils/functions/StringUtils.ts

```diff
@@ -75,3 +75,25 @@ export function validatePassword(newPassword: string) {
   }
   return undefined
 }
+
+export function copyStringToClipboard(
```

Comment:
Is this where this util function should live?

Reply:
in general we could use a better system for organizing utils

Reply:
If you did this, would you have to change the API?

```ts
export function copyStringToClipboard(value: string): Promise<void> {
  return new Promise((resolve, reject) => {
    // PORTALS-3571: setTimeout necessary on Safari
    setTimeout(() => {
      // use the Clipboard API
      // https://caniuse.com/mdn-api_clipboard_writetext
      navigator.clipboard.writeText(value).then(resolve).catch(reject)
    }, 0)
  })
}
```